lib/commands/autocomplete/helper.js

```diff
@@ -0,0 +1,61 @@
+const fs = require('fs-extra');
+const path = require('path');
+
+module.exports = class Helper {
+    constructor(omelette, commanders = []) {
+        this.commanders = commanders;
+        this.completion = omelette('ask');
+        this.autoCompleteHintsFile = path.join(__dirname, 'autocomplete-hints.json');
+    }
+
+    _getAutoCompleteOptions() {
```

Comment:
minor, should this func be renamed to `_getAutoCompleteCmd`?

Reply:
why cmd? it just loads auto complete options tree.
{
 smapi: [commandOne, commandTwo...]
...
}